Postgresql opentelemtry app enhanced with newly contributed metrics (#5287)
SumoLogic/sumologic-documentation: docs/integrations/databases/opentelemetry/postgresql-opentelemetry.md

- Monitor PostgreSQL relation metrics (disk blocks, buffer hits, and hot updates) trends over time.
- Monitor index scans and size to determine if executed queries are accessing them for a relation.
- Track index utilization of existing indexes in a relation.

> - Monitor the Postgresql live and dead rows count trend.

Reviewer: ```suggestion
- Monitor the PostgreSQL live and dead rows count trend.
```

| `PostgreSQL - Instance Down Alert` | This alert gets triggered when the Postgres instance is down. | Count > = 1 | Count < 1 |
| `PostgreSQL - SlowQueries Alert` | This alert gets triggered when we detect that the PostgreSQL instance is executing slow queries. | Count > 5 | Count < = 5 |
| `Postgresql- Too Many Connections Alert` | PostgreSQL instance has too many connections. | Count > = 100 | Count < 100 |

> | `Postgresql - Too Many Locks Acquired Alert` | This alert is triggered when there are too many locks acquired on the database. Increase the postgres setting `max_locks_per_transaction`, if this alert occurs frequently. | Count > = 100 | Count < 100 |
> | `Postgresql- High Rate Deadlock` | This alert is triggered when deadlocks in a Postgres instance are detected. | Count > = 1 | Count < 1 |

Reviewer: ```suggestion
| `PostgreSQL - Too Many Locks Acquired Alert` | This alert is triggered when there are too many locks acquired on the database. Increase the postgres setting `max_locks_per_transaction`, if this alert occurs frequently. | Count > = 100 | Count < 100 |
| `PostgreSQL - High Rate Deadlock` | This alert is triggered when deadlocks in a Postgres instance are detected. | Count > = 1 | Count < 1 |
```

| `PostgreSQL - Instance Down Alert` | This alert gets triggered when the Postgres instance is down. | Count > = 1 | Count < 1 |
| `PostgreSQL - SlowQueries Alert` | This alert gets triggered when we detect that the PostgreSQL instance is executing slow queries. | Count > 5 | Count < = 5 |
| `Postgresql- Too Many Connections Alert` | PostgreSQL instance has too many connections. | Count > = 100 | Count < 100 |

> | `Postgresql - Too Many Locks Acquired Alert` | This alert gets triggered when we detect that there are too many locks acquired on the database. If this alert happens frequently, you may need to increase the postgres setting `max_locks_per_transaction`. | Count > = 100 | Count < 100 |

Reviewer 2: ```suggestion
| `Postgresql - Too Many Locks Acquired Alert` | This alert is triggered when there are too many locks acquired on the database. If this alert happens frequently, you may need to increase the postgres setting `max_locks_per_transaction`. | Count > = 100 | Count < 100 |
```

> | `Postgresql- High Rate Deadlock` | This alert fires when we detect deadlocks in a Postgres instance | Count > = 1 | Count < 1 |

Reviewer 2: ```suggestion
| `Postgresql- High Rate Deadlock` | This alert is triggered when deadlocks in a Postgres instance are detected. | Count > = 1 | Count < 1 |
```
